### DISCUSSION TITLE
Blueprint problem

### DISCUSSION THREAD

**Discussion by Stasky1968** [HASwitchPlate/HASPone#102] [Q&A]
I have some issue wit some blueprint
I have install a new plate
"The blue print for HASP Core functionality" works ok
"HASP activates a selected page after a specified period of inactivity" work ok
"HASP p[x].b[y] activates a page" work ok
HASP p[x].b[y] displays a clock with a clock icon - HASP p[x].b[y] displays a toggle button - HASP p[x].b[y] displays the current temperature with icon and colors 
Not work
Enclose the trace automation for HASP p[x].b[y] displays the current temperature with icon and colors
[trace automation.hasp_p_x_b_y_displays_the_current_temperature_with_icon_and_colors 2021-05-11T17_44_54.897600+00_00.txt](https://github.com/HASwitchPlate/HASPone/files/6461068/trace.automation.hasp_p_x_b_y_displays_the_current_temperature_with_icon_and_colors.2021-05-11T17_44_54.897600%2B00_00.txt)

**Comment by aderusha** [comment]
edit: [Check out this wiki entry](https://github.com/HASwitchPlate/HASPone/wiki/Fix-common-HASPone-issues) and see if that resolves the issues you are seeing!

**Reply by Stasky1968** [reply]
Thanks so much for your help, now it works.
Many compliments for this project
